src/onegov/org/forms/event.py

```diff
@@ -653,3 +653,20 @@ class EventConfigurationForm(Form):
             'class_': 'formcode-select',
             'data-fields-include': 'radio,checkbox'
         })
+
+    def validate(self):
+        """
+        Ensures none of the active keywords is used in the `EventForm` to
+        prevent overriding data accidentally by defining filters.
+        """
+        result = super().validate()
+
+        for keyword in [k.lower() for k
+                        in self.keyword_fields.data.splitlines()]:
+            if keyword in dir(EventForm):
+                error = f'Keyword \'{keyword}\' is already in use!'
```

Comment:
```suggestion
                error = self.definition.gettext(_(
                    'Field name \'${name}\' is already in use!',
                    mapping={'name': keyword})
                ))
```
i18n support

Reply:
Also custom validators are generally defined in a separate function prefixed by `ensure_` you don't need to override the `validate` method.

Reply:
Also does `display` actually matter? Shouldn't `definition` be merged into `EventForm` regardless? I think you only really need to check definition, in which case it makes more sense to put this validation into `ValidFilterFormDefinition`

Reply:
@Tschuppi81 Actually `ValidFormDefinition` already has `reserved_fields` arguments, so you can override `__init__` in `ValidFilterForm` to call `self.reserved_fields.update(name for name, __ in EventForm._unbound_fields)`. You will need to move the validator from `onegov.form` to `onegov.org` which makes more sense to me anyways, since you are only using this validator there.

Reply:
Ok, I' ll try

Reply:
The `reserved_fields` of `ValidFormDefinition` did all I needed!